tests/materials/processing/strategies/test_aggregates_strategy.py

```diff
@@ -43,18 +47,20 @@ def test_gather_composition_properties_adds_all_properties():
     )
 
     result = AggregatesStrategy.gather_composition_information(aggregates)
-    assert result == ['Fine Aggregates (kg/m³): 123.45, ',
-                      'Coarse Aggregates (kg/m³): 67.89, ',
-                      'FA Density (kg/m³): 987.6, ',
-                      'CA Density (kg/m³): 543.2, ']
+    assert result == ['Fine Aggregates (m%): 123.45, ',
+                      'Coarse Aggregates (m%): 67.89, ',
+                      'Specific Density (kg/m³): 987.6, ',
```

Comment:
Im Ux sollte das auch Specific Gravity und nicht Specific Density heißen. Das muss in der entspechenden gather_composition_information Methode auch berücksichtigt werden